src/services/shims.ts

```diff
@@ -467,7 +467,7 @@ namespace ts {
         }
     }
 
-    export class CoreServicesShimHostAdapter implements ParseConfigHost, ModuleResolutionHost {
+    export class CoreServicesShimHostAdapter implements ParseConfigHost, ModuleResolutionHost, JsTyping.TypingResolutionHost {

```

Comment:
It always implemented that interface. TypeScript's structural type system doesn't require us to declare these things, but it doesn't hurt.